dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -62,34 +65,45 @@
   private static void setDetectedOsSystemProperties() {
     // System properties to select Netty dependencies through os-maven-plugin
     // Definition of the properties: https://github.com/trustin/os-maven-plugin
-    String nonAlphaNumericPattern = "[^a-z0-9]+";
+
+    System.setProperty("os.detected.name", osDetectedName());
+
+    System.setProperty("os.detected.arch", osDetectedArch());
+
+    System.setProperty("os.detected.classifier", osDetectedName() + "-" + osDetectedArch());
```

Comment:
Instead of recalculating these two, maybe just use System.getProperty()?

Reply:
Introduced variables for them.